docs/user_guide/perf_analyzer.md

```diff
@@ -26,716 +26,6 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 -->
 
-# Performance Analyzer
+Perf Analyzer documentation has been relocated to here:
```

Comment:
This might be better with a link here instead of the full URL. I can see the appeal of including the whole URL, but it deviates from the approach used in most of Triton documentation.

Reply:
Used inline link 👍